### ISSUE TITLE
Transformer model

### ISSUE THREAD

**Issue by Egbohou-Kevin** [rivercold/BERT-unsupervised-OOD#8]
Hello!
Is it possible to use another model like CamemBERT using load_path argument? or it is only for fine-tuned Bert and Roberta model?
Thanks in advance!

**Comment by rivercold** [comment]
Yes, it is possible to use another alternative BERT model by switching to  `AutoModelForSequenceClassification`.  But my current implementations are hard-coded for Bert and Roberta, you may need to make some changes for a few functions, e.g., how to extract hidden representations from the pre-trained LM:
https://github.com/rivercold/BERT-unsupervised-OOD/blob/9e1cd4c06ec3273563cbbffe431d31971f153ca0/our_model.py#L256.

**Comment by Egbohou-Kevin** [comment]
thank you.
I have another question about DTACC. In article we have no information about what it correspond to and how is it compute. I want to know it is like normal accuracy or not. Thank you

**Comment by rivercold** [comment]
The code for computing DTACC is https://github.com/rivercold/BERT-unsupervised-OOD/blob/9e1cd4c06ec3273563cbbffe431d31971f153ca0/calculate_log.py#L89. 

Besides, you can check Section 4.3 https://arxiv.org/pdf/1706.02690.pdf for the definition of detection error, which is  1- DTACC.

**Comment by rivercold** [comment]
I am closing this issue. Feel free to create a new one if you have any other questions!
